app/lib/navigation/state_file_base_question_navigation.rb

```diff
@@ -1,11 +1,14 @@
 module Navigation
-  module StateFileBaseQuestionNavigationMixin
+  class StateFileBaseQuestionNavigation
+    class << self
 
-    def self.included(base)
-      base.extend(ClassMethods)
-    end
+      def navigation_class_from_state_code(state_code)
+        unless StateFile::StateInformationService::STATES_INFO.key?(state_code)
+          raise StandardError state_code
```

Comment:
This might not be in scope for this story, but it bothers me a bit that the message here is just the state_code, and it occurs to me that it would make more sense to create an `InvalidStateCodeError` class that can be used where needed with a clearer error.